src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -39,13 +40,17 @@ protected function configure()
                 new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
                 new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of places)'),
                 new InputOption('label', 'l', InputArgument::OPTIONAL, 'Labels a graph'),
+                new InputOption('dump_format', null, InputOption::VALUE_REQUIRED, 'The dump format [dot|puml]', 'dot'),
```

Comment:
I know I tell you to use `dump_format` but actually It should  be `dump-format` (dash instead of underscore)
And It's the same for `puml_transition_format`

Sorry :/

Reply:
done.